example-crates/wallet_rpc/src/main.rs

```diff
@@ -86,25 +91,25 @@ fn main() -> anyhow::Result<()> {
     );
 
     let start_load_wallet = Instant::now();
-    let mut db = Store::<bdk_wallet::wallet::ChangeSet>::open_or_create_new(
-        DB_MAGIC.as_bytes(),
-        args.db_path,
-    )?;
-    let changeset = db.aggregate_changesets()?;
+    let conn = Connection::open_in_memory().expect("must open connection");
+    let mut db = Store::new(conn).expect("must create db");
+    let changeset = db.read()?;
+    let external_descriptor = args.descriptor.as_str();
+    let internal_descriptor = args.change_descriptor.as_str();
 
     let mut wallet = Wallet::new_or_load(
-        &args.descriptor,
-        &args.change_descriptor,
+        external_descriptor,
+        internal_descriptor,
         changeset,
-        args.network,
+        Network::Signet,
```

Comment:
We still want to get the network from the CLI args since it's configurable

```suggestion
        args.network,
```